src/dataset.hpp

```diff
@@ -70,7 +76,7 @@ sparse_dataset<T>::sparse_dataset(const std::filesystem::path &dir_path)
 		
 		if (entry.path().extension() == ".attr")
 		{
-			LOG << "found attribute file - " << entry << "\n";
+			//LOG << "found attribute file - " << entry << "\n";
```

Comment:
Logowanie kontroluje się makrem `DEBUG_LOGGING`. Wystarczy kompilować w trybie release i tego nie będzie.

```suggestion
			LOG << "found attribute file - " << entry << "\n";
```

Reply:
done